InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -41,6 +42,9 @@ def __init__(self, **kwargs: Any) -> None:
             pooling_type=GatedAttentionLayer.__name__,
             # average number of tiles is 56 for PANDA
             encoding_chunk_size=60,
+            cache_mode=CacheMode.MEMORY,
+            precache_location=CacheLocation.SAME,
+            batch_size=8,
```

Comment:
Is batch_size 8 the maximum we can handle during fine-tuning? it seems that the set of hyperparameters we will need to use for DeepMIL with fine tune and without will be pretty different. If we want to leave track of this in the form of configs file in the repo one option is to start defining some child classes, for example one for DeepSMILEPandaFrozen and one for DeepSMILEPandaTuning, where in each we overwrite the parameters of this constructor that needs to be changed

Reply:
Thanks. I will remove this line to keep the original batch size. For me, batch size 8 worked but it will require more experimentation to determine the optimal hyperparameters for finetuning. I would not create the new class as of now, but we could revisit this when we are experimenting more with finetuning. In this PR, I just want to enable the fine-tuning and we could optimize hyperparameters in the specific experiment

Reply:
I have added the hyperparameters for which fine-tuning works on PANDA in the comments, as suggested earlier